modules/ROOT/pages/design-edit-mulitple-tabs.adoc

```diff
@@ -0,0 +1,37 @@
+= Edit in Multiple Tabs in the Text Editor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can have multiple files open at a time in API Designer's text editor.
+
+== Opening Files in New Tabs
+
+There are four ways that you can open a file in a new tab:
+
+* Open the context menu for a file in the file tree by clicking the three dots to its right, and then select *Open in new tab*.
+* Press the Command key (MacOS) or the Control key (Windows) and click the name of the file in the file tree.
+* Place your cursor in the name of a file for an `include` tag. Then, press the Command key (MacOS) or the Control key (Windows) and click the filename.
```

Comment:
This is true not only for files referred with include, but also with **use** when referring to libraries and also with $ref in OAS Apis, we need to mention opening files from references in general